docs/content/Workspace/Auto-Suspend.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Auto-suspend
+permalink: /workspace/auto-suspend
+category: Workspace
+menuOrder: 3
+---
+
+<SuccessBox>
+
+Auto-suspend is available in Cube Cloud on
+[Starter and above tiers](https://cube.dev/pricing).
+
+</SuccessBox>
+
+Cube Cloud can automatically suspend infrastructure when not in use to prevent
+CCU consumption when resources are not being actively used. This is useful for
```

Comment:
I don't believe it's clear from the get-go what "CCU consumption" is. I think it might be better to be more high-level here and say something less specific than "CCU". Maybe "resource consumption" (then "resources are not being used" would use "infrastructure"), "overspend", "unnecessary quota use", etc.

Reply:
Also, having a link to the pricing page would help, right? The one where CCUs are defined.